modules/basics/pages/system-and-language-basics.adoc

```diff
@@ -65,6 +67,50 @@ To exit the GSQL shell, type either `exit` or  `quit` at the GSQL prompt:
 
 `GSQL> EXIT`  or `GSQL> QUIT`
 
+=== New token-based authentication method
+
+Starting with version *4.1.0*, GSQL now supports token-based authentication, allowing users to log into the GSQL shell using a token instead of a traditional username and password.
+
+*How to login using a token*
+
+To log into the GSQL shell using a token, you can use the `-t` or `--token` option, followed by the token value. The token can either be a plain-text token or a JWT token.
```

Comment:
```suggestion
To log into the GSQL shell using a token, use the `-t` or `--token` option, followed by the token value. The token can either be a plain-text token or a JWT token.
```
Do not say "you can" when telling the user what they *must* do to do something properly. Use the imperative mood: "Open the door", not "You can open the door."
This is a common grammatical error that our developers make. Don't copy their mistake.
However, "can" is ok for "plain-text token or a JWT token" because there the user has a choice.